ui/site/src/component/assets.ts

```diff
@@ -7,18 +7,24 @@ export const assetUrl = (path: string, opts: AssetUrlOpts = {}) => {
   return baseUrl + '/assets' + (opts.noVersion ? '' : '/_' + version) + '/' + path;
 };
 
-const loadedCss = new Map<string, true>();
-export const loadCss = (url: string) => {
+const loadedCss = new Map<string, Promise<void>>();
```

Comment:
Made some changes here such that we load CSS files with promises so that we can await them, similar to how we load other things in this file